OWNERS_ALIASES

```diff
@@ -8,3 +8,11 @@ aliases:
   sig-critical-approvers-parser:
     - yudongusa
     - easonn7
+  sig-approvers-autoid-service: # approvers for auto-id service
```

Comment:
Maybe we can just need 1 owner only: @tiancaiamao

Reply:
you can update it by suggestion codes.

> Maybe we can just need 1 owner only: @tiancaiamao

Reply:
It is best to set up 2 or more members.